src/platform/i386/chal/chal_pgtbl.h

```diff
@@ -0,0 +1,5 @@
+extern int chal_pgtbl_kmem_act(pgtbl_t pt, u32_t addr, unsigned long *kern_addr, unsigned long **pte_ret);
```

Comment:
Why are these all `extern`?

Why don't you have "include once" guards on this file?

Reply:
The guard is added.